spec/puppet-lint/plugins/check_classes/arrow_on_right_operand_line_spec.rb

```diff
@@ -11,7 +11,8 @@
 
       context 'arrow on the line of left operand' do
         let(:code) do
-          "Package['httpd']  #{operator}
+          "
+            Package['httpd']  #{operator}
```

Comment:
That's a good question, 10.4 doesn't say how it should be aligned. @jbondpdx any thoughts?